tests/components/tasmota/test_common.py

```diff
@@ -304,16 +335,17 @@ async def help_test_entity_id_update_discovery_update(hass, mqtt_mock, domain, c
     async_fire_mqtt_message(hass, f"{DEFAULT_PREFIX}/{config[CONF_MAC]}/config", data)
     await hass.async_block_till_done()
 
-    async_fire_mqtt_message(hass, topic, get_state_online(config))
+    async_fire_mqtt_message(hass, topic, config_get_state_online(config))
     state = hass.states.get(f"{domain}.test")
     assert state.state != STATE_UNAVAILABLE
 
-    async_fire_mqtt_message(hass, topic, get_state_offline(config))
+    async_fire_mqtt_message(hass, topic, config_get_state_offline(config))
     state = hass.states.get(f"{domain}.test")
     assert state.state == STATE_UNAVAILABLE
 
     entity_reg.async_update_entity(f"{domain}.test", new_entity_id=f"{domain}.milk")
     await hass.async_block_till_done()
+    assert hass.states.get(f"{domain}.milk")
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)